contrib/pipelines/secadvisor/main.go

```diff
@@ -46,7 +46,7 @@ func NewSubscriberFromConfig(cfg *viper.Viper) (*websocket.StructSpeaker, error)
 		return nil, fmt.Errorf("Cannot initialize classify: %s", err)
 	}
 
-	encode, err := core.NewEncodeJSON()
+	encode, err := core.NewEncodeFromConfig(cfg)
```

Comment:
I suggest adding the following to `contrib/pipelines/secadvisor/secadvisor.yml.default`:

```diff
diff --git a/contrib/pipelines/secadvisor/secadvisor.yml.default b/contrib/pipelines/secadvisor/secadvisor.yml.default
index edc37c7..0710540 100644
--- a/contrib/pipelines/secadvisor/secadvisor.yml.default
+++ b/contrib/pipelines/secadvisor/secadvisor.yml.default
@@ -41,3 +41,7 @@ pipeline:
       max_seconds_per_object: 60
       max_seconds_per_stream: 86400
       max_flow_array_size: 100000
+  encode:
+    type: json
+    json:
+      pretty: true
```

Because this change causes the the `secadvisor` pipeline to panic when used with the default config file.

Reply:
addressed in #1923